api/checks/webhook.go

```diff
@@ -266,19 +268,10 @@ func createCheckRun(ctx context.Context, suite shared.CheckSuite, opts github.Cr
 		return false, err
 	}
 	client := github.NewClient(jwtClient)
-	u := fmt.Sprintf("repos/%v/%v/check-runs", suite.Owner, suite.Repo)
-	req, err := client.NewRequest("POST", u, opts)
-	if err != nil {
-		return false, err
-	}
-	req.Header.Set("Accept", "application/vnd.github.antiope-preview+json")
-	checkRun := new(github.CheckRun)
-	resp, err := client.Do(ctx, req, checkRun)
 
+	checkRun, resp, err := client.Checks.CreateCheckRun(ctx, suite.Owner, suite.Repo, opts)
 	if err != nil {
 		log.Warningf("Failed to create check_run: %s", resp.Status)
-		body, _ := ioutil.ReadAll(resp.Body)
-		log.Warningf(string(body))
```

Comment:
Is this logging not useful?

Reply:
No, it was a debugging effort that didn't get cleaned up.